src/openms/source/ANALYSIS/MAPMATCHING/QTClusterFinder.cpp

```diff
@@ -143,49 +232,58 @@ namespace OpenMS
     for (list<QTCluster>::iterator it = clustering.begin();
          it != clustering.end(); ++it)
     {
-      OpenMSBoost::unordered_map<Size, GridFeature*> elements;
-      typedef std::multimap<double, GridFeature*> InnerNeighborMap;
-      typedef OpenMSBoost::unordered_map<Size, InnerNeighborMap> NeighborMap;
-      NeighborMap neigh = it->getNeighbors();
-      for (NeighborMap::iterator n_it = neigh.begin(); n_it != neigh.end(); 
-           ++n_it)
+      typedef OpenMSBoost::unordered_map<Size, std::vector<GridFeature*> > NeighborMap;
```

Comment:
"NeighborMap" denotes two different types in QTClusterFinder and in QTCluster. Could that lead to confusion?